web/src/buddy_data.ts

```diff
@@ -1,8 +1,5 @@
-import _ from "lodash";
 import assert from "minimalistic-assert";
 
-import * as blueslip from "./blueslip.ts";
-import {ConversationParticipants} from "./conversation_participants.ts";
```

Comment:
Does `web/src/conversation_participants.ts` end up being dead code after this? We should probably delete it if so.

Reply:
Ahh, no, it's still used in `message_list_data.ts`.

Reply:
Yup.